apps/web/app/(authenticated)/(app)/new/keys.tsx

```diff
@@ -78,7 +78,7 @@ export const Keys: React.FC<Props> = ({ apiId }) => {
 
   const createKeySnippet = `curl -XPOST '${
     process.env.NEXT_PUBLIC_UNKEY_API_URL ?? "https://api.unkey.dev"
-  }/v1/keys' \\
+  }/v1/keys.verifyKey' \\
```

Comment:
@perkinsjr holdup
this is `/v1/keys.createKey`

Reply:
Copy pasta.